docs/process.md

```diff
@@ -2,59 +2,83 @@
+going on before, during, and after this scope.
 
-New work is proposed via [Issues](https://github.com/mozilla/testpilot/issues/new).
-
-## Estimation
-
-Please add a size to all new issues.
-
-- `size:small` ~1 hour
-- `size:medium` ~1 day
-- `size:large` ~1 week
-- `size:too large` >1 week
+## Waffle
 
-Estimation is an imperfect science, so please just give your best guess. If you're completely unsure of sizing, apply the `needs:size` label.
+Test Pilot uses a [waffle board](https://waffle.io/mozilla/testpilot) to
+organize issues.  The columns are:
 
-## Current Milestone
+* *Untriaged*:  These are new issues we haven't looked at yet.
+* *Backlog*:  We've looked at these issues, but there are still some open
+  questions or were not ready to start working on them yet.
+* *Ready*:  These issues have been triaged, questions are answered, assets are
+  available, and work is ready to begin.  To be in this column an issue *must*
+  have a `size` label.
+* *In Progress*:  These issues are being actively worked on.  To be in this
+  column an issue must be assigned to someone.  PRs also land here.
+* *Done*: These issues are closed.  That likely means a fix was committed, but
+  it may mean an alternative was found or we decided to wontfix the issue.
 
-The current milestone is packed every 2 weeks, and coincides with the Friday before bi-weekly planning sessions. Planning sessions happen every 2nd Monday.
+An issue marked *done* is not necessarily live on the site.  Pushes are done
+periodically (more details below).
 
-The milestones are planned in [waffle.io](https://waffle.io/mozilla/testpilot/), and a linked label is added in GitHub: [status:planned](https://github.com/mozilla/testpilot/issues?q=is%3Aissue+is%3Aopen+label%3A%22status%3A+planned%22).
+## New Work
 
-## Status
+New work is proposed via [Issues](https://github.com/mozilla/testpilot/issues/new).
 
-- `status:ready`
+## Triage
 
-This label is applied when all UX (`needs UX`), product and other discussions (`needs discussion`) have been sorted out.
+Generally, triage will review the *Untriaged* column on the Waffle board.  If
+an issue isn't clear, complete, or has dependencies:
+* Ask the question in the issue and @-tag the person who can help
+* Use the `needs:discussion`, `needs:moreinfo`, `needs:product`, `needs:qa`,
+  and `needs:ux` labels
+* Move the issue to *Backlog*
 
-This is linked to the `Ready` column on our waffle board.
+If the issue is ready to be worked on:
+* Assign an appropriate `size` label
+* Consider any other labels (perhaps it's a `good-first-bug`?)
+* Move the issue to *Ready*
 
-- `status:planned`
+## Labels
 
-This label is applied to work we have identified in our planning meeting as targeting our current milestone.
+Size labels are roughly defined as:
 
-This is linked to the `Current Milestone` column on our waffle board.
+- `size:small` ~1 hour
+- `size:medium` ~1 day
+- `size:large` ~1 week
+- `size:too large` >1 week
```

Comment:
I don't know how important/necessary sizing becomes in Kanban. Given we aren't loading a finite amount of time.